src/charm.py

```diff
@@ -169,7 +169,7 @@ def _relation_created(self, relation_name: str) -> bool:
         Returns:
             bool: Whether the relation was created.
         """
-        return bool(self.model.get_relation(relation_name))
+        return len(self.model.relations[relation_name]) >= 1
```

Comment:
We can keep the `bool`, it will be `True` as long as the list is not empty.
```suggestion
        return bool(self.model.relations[relation_name])
```